### ISSUE TITLE
Muted players need to have limited access to features.

### ISSUE THREAD

**Issue by AiAe** [Quaver/Z#10]
When muted players should not be able to:
1. Play multiplayer (and invite)
2. Spectate players

**Comment by Swan** [comment]
I don't agree with this at all. If they're abusive in chat, then they should just not be allowed to chat.

**Comment by AiAe** [comment]
If you don't want to do this then I would prefer if we disable any kind of Player interaction then, like invites to MP game. 
Perfect example from today, muted player for spamming players, then he went to MP game and spams them with invites.

**Comment by AiAe** [comment]
We either need to implement something on server side (which would be easier) or allow in Client player to ignore invites/dms from non friends or something like that (or preferably just mute specific players)

**Comment by Swan** [comment]
Better off keeping it simple and just adding a block list.
